geonetwork/content.md

```diff
@@ -12,50 +12,130 @@ The project is part of the Open Source Geospatial Foundation ( [OSGeo](http://ww
 
 %%LOGO%%
 
-# How to use this image
+## How to use this image
+
+GeoNetwork 4 uses an Elasticsearch server to store the index of the documents it manages so **it can't be run without configuring the URL of the Elasticsearch server**.
+
+### Elasticsearch configuration
+
+#### Java properties (since version 4.4.0)
```

Comment:
Version 4.4.x is not the recommended "stable" release - most readers will use the first example.

Is this README associated with a specific release? Or is it general purpose.

Reply:
It's general.